### PR TITLE
[fr] consistency: jack Jack JACK

### PR DESCRIPTION
**Pull request by trebmuh**
JACK is an acronym, a recursive one, which mean JACK Audio Connection Kit. Hence is should always be in capital letters.

### CODE REVIEW

File: wiki/fr/fr-Installation-for-Linux.md [hunk 1]
```diff
@@ -74,27 +74,27 @@ sudo make install
...
 
 ### Démarrer Jamulus
 
 Ouvrez Jamulus, par ex. en ligne de commande. Si vous utilisez la ligne de commande, exécutez la commande `Jamulus` (avec un « J » majuscule) qui démarre le client Jamulus.
 
 Jamulus se trouve dans `/usr/local/bin`. Vous pouvez maintenant supprimer le répertoire des sources que vous venez de compiler.
 
-Si vous avez des problèmes de son qui se coupe (en particulier les erreurs XRUN rapportées par Jack/QJackCtl), essayez de définir des valeurs plus élevées (par exemple 256 échantillons ou 3 périodes) à l'étape 3 ci-dessus. Des valeurs plus basses (par ex. 64 échantillons) pourraient apporter de meilleures performances mais peut-être plus de problèmes avec le son. Sinon, consultez la [section dépannage](Client-Troubleshooting).
+Si vous avez des problèmes de son qui se coupe (en particulier les erreurs XRUN rapportées par JACK/QJackCtl), essayez de définir des valeurs plus élevées (par exemple 256 échantillons ou 3 périodes) à l'étape 3 ci-dessus. Des valeurs plus basses (par ex. 64 échantillons) pourraient apporter de meilleures performances mais peut-être plus de problèmes avec le son. Sinon, consultez la [section dépannage](Client-Troubleshooting).
```
Comment (ann0see) [inline wiki/fr/fr-Installation-for-Linux.md:97]: I think we also discussed to use "QjackCtl" not "QJackCtl"

**Review by ann0see — DISMISSED** [review]

**Comment by ann0see** [comment]
All in all looks good.

File: wiki/fr/fr-Installation-for-Linux.md (hunk 1 above)
Comment (trebmuh) [reply wiki/fr/fr-Installation-for-Linux.md:97]: This PR is about jack/Jack/JACK.
I've got a bunch of other consistency PR to make, but I prefer to look at these one by one.
Please, merge this one as is.

**Comment by trebmuh** [comment]
@ann0see please, merge.

File: wiki/fr/fr-Installation-for-Linux.md (hunk 1 above)
Comment (ann0see) [reply wiki/fr/fr-Installation-for-Linux.md:97]: Ah. Ok

**Review by ann0see — APPROVED** [review]

File: wiki/fr/fr-Installation-for-Linux.md (hunk 1 above)
Comment (trebmuh) [reply wiki/fr/fr-Installation-for-Linux.md:97]: thank you very much

**Pull request merged**